src/components/board-game/index.jsx

```diff
@@ -2,7 +2,12 @@ import React, {useEffect, useRef, useState} from 'react';
 import Card from "../card";
 import '../../styles/variables.css';
 import styles from './Board.module.css';
-import {parsingShuffles, createPrimeArray, shuffle} from "../../utils";
+import {constants} from 'shares/constants';
+import classnames from "classnames"
```

Comment:
Use `camelCase` for such names
```suggestion
import classNames from "classnames"
```